.github/workflows/dotnetcore.yml

```diff
@@ -19,35 +19,6 @@ jobs:
         dotnet-version: 5.0.402
     - name: Install dependencies
       run: dotnet restore
-    - name: Build
```

Comment:
better not forget this

Reply:
Don't worry, the plan is to revert these CI files when I have all 4 libs working.